tokio/src/runtime/enter.rs

```diff
@@ -104,6 +105,42 @@ cfg_blocking_impl! {
                 park.park()?;
             }
         }
+
+        /// Blocks the thread on the specified future, returning the value with
+        /// which that future completes.
```

Comment:
Minor nit: looks like the comment was copied from `block_on` and doesn't note anything about the timeout.